akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -98,6 +98,16 @@ object Sink {
   def headOption[T]: Sink[T, Future[Option[T]]] = new Sink(new HeadOptionSink[T](DefaultAttributes.headSink, shape("HeadOptionSink")))
 
   /**
+   * TODO: document usage
+   *
+   */
+  def toSeq[T]: Sink[T, Future[Seq[T]]] = {
```

Comment:
I don't like the `toSeq` name: `Source(1 to 10) via flow to Sink.toSeq` <-- toto?
There are no other Sink starts with `to` so there's no prior art to refer to.